src/core_plugins/kibana/server/tutorials/apache_logs/index.js

```diff
@@ -13,15 +13,15 @@ export function apacheLogsSpecProvider() {
                      ' [Learn more]({config.docs.beats.filebeat}/filebeat-module-apache2.html)' +
                      ' about the apache2 module.',
     //iconPath: '', TODO
-    /*artifacts: {
-      dashboards: [
-        {
-          id: 'Filebeat-Apache2-Dashboard',
-          linkLabel: 'Apache2 logs dashboard',
-          isOverview: true
-        }
-      ]
-    },*/
+    artifacts: {
+      actionButton: {
+        url: '#/dashboard/Filebeat-Apache2-Dashboard',
```

Comment:
My only concern here is that Kibana might internally change their URL structure and then all the tutorials that specify such URLs might need to be updated. What do you think about leaving the `dashboards` section as-is but also including a more-generic `actionButton` section?

Reply:
I think we already rely on relative URLs in the Beats and Logstash dashboards (when they cross-link each other).

I agree to keep the `dashboards` section. This wasn't meant to replace it, since at some point it's going to be useful for Kibana to know which dashboards are added by which module. But the button from the bottom should be based on the `actionButton` section, right?

Reply:
Maybe the actionButton can be optional and used to override the default behavior of linking to the target dashboard?

Reply:
That works for me. Perhaps it's best if you adjust the Joi schema as you implement the feature?

Reply:
sounds good. Are you going to add the artifact stuff or should I.

Reply:
Can you do it please? I’m just about to start my traveling, so I won’t be able to do it too soon.

Reply:
Will do. Have a great vacation.